docs/package_integration/uds-package-practices.md

```diff
@@ -1,110 +1,102 @@
   > This allows for different images or configurations to be delivered consistently to customers.
+- **May** template network policy keys to provide flexibility for delivery customers to configure.
+- **May** end any generated secrets with `-sso` to easily locate them when querying the cluster.
+- **May** template Keycloak fields to provide flexibility for delivery customers to configure.
 
-## Testing
-
-A UDS Package will also have testing and quality checks to ensure that updates / changes to them result in minimal churn.  Packages:
+## Bronze
 
-- **Must** implement Journey Testing to cover the basic user flows and features of the application, especially where an application interacts with an external service / interface.
-  > This can by something like Playwright / [Cypress](https://github.com/defenseunicorns/uds-identity-config/tree/main/src/test/cypress) tests for services with a Web UI or something like [Jest](https://github.com/defenseunicorns/uds-package-gitlab-runner/tree/main/test) tests for headless services.
+_a Bronze UDS-Package that meets the minimum requirements and becomes compatible but not optimal or fully integrated with UDS. It is not ready to run in production without significant caveats._
 
-- **Must** implement Upgrade Testing to ensure that the current development package works when deployed over the previously released one.
+Bronze packages:
 
+- **Should** be created from the [UDS Package Template](https://github.com/defenseunicorns/uds-package-template)
+- **Must** be declaratively bundled in a [Zarf package](https://docs.zarf.dev/ref/create/)
+- **Must** define any external interfaces under the `expose` key in the [UDS Package Custom Resource](https://github.com/defenseunicorns/uds-core/blob/main/docs/configuration/uds-operator.md)
+- **Must** deploy and operate successfully with Istio injection enabled in the namespace.
+- **Must** implement Journey testing, covering the basic user flows and features of the application (see [Testing Guidelines](./testing-guidelines.md))
+  > This can by something like Playwright / [Cypress](https://github.com/defenseunicorns/uds-identity-config/tree/main/src/test/cypress) tests for services with a Web UI or something like [Jest](https://github.com/defenseunicorns/uds-package-gitlab-runner/tree/main/test) tests for headless services.
+- **Must** implement Upgrade Testing to ensure that the current development package works when deployed over the previously released one. (see [Testing Guidelines](./testing-guidelines.md))
+- **Must** be capable of operating within an internet-disconnected (air-gapped) environment
+- **Must** be actively maintained by the package maintainers identified in CODEOWNERS
+- **Must** be versioned using the UDS Package [Versioning scheme](#versioning)
+- **Must** contain documentation under a `docs` folder at the root that describes how to configure the package and outlines package dependencies.
```

Comment:
Maybe just my OCD but do we want to still have sub-categories under the levels (like `UDS Core Integrations`, `Package Structure`, and `Maintenance Practices`?)

Reply:
Yes. Created https://github.com/defenseunicorns/uds-common/issues/255 are you okay defering these changes to that issue? I'm not 100% the best way to organize the information to retain conciseness and provide that level of clarification.

Reply:
https://github.com/defenseunicorns/uds-common/issues/255

Reply:
I think that's fine to capture in that issue for now.